templates/lxd/index.html

```diff
@@ -0,0 +1,299 @@
+        <li class="p-list__item is-ticked">LTS releases every two years &mdash; supported for five years</li>
+        <li class="p-list__item is-ticked">Commercial support available through <a href="https://ubuntu.com/pro">Ubuntu Pro</a></li>
+      </ul>
+    </div>
+  </div>
+</section>
+
+<section class="p-section--shallow">
+  <hr class="p-rule is-fixed-width">
+  <div class="u-fixed-width">
+    {{ image (
+      url="https://assets.ubuntu.com/v1/ca68d7e3-stocksy-woman-working-ar2-1.jpg",
+      alt="",
+      width="2048",
+      height="836",
+      hi_def=True,
+      loading="lazy"
+      ) | safe
+    }}
+  </div>
+</section>
+
+<section class="p-section">
+  <hr class="p-rule is-fixed-width">
+  <div class="row--50-50">
+    <div class="col">
+      <h3 class="p-heading--2">Join the LXD community</h3>
+    </div>
+    <div class="col">
+      <div class="row col-medium-2">
+        <div class="col-2 col-small-2">
+          <h3 class="p-text--small-caps u-no-margin--bottom">Documentation</h3>
+        </div>
+        <div class="col-4 col-small-2">
+          <p><a href="https://documentation.ubuntu.com/lxd/en/latest/">Read the documentation&nbsp;&rsaquo;</a></p>
+        </div>
+      </div>
+      <hr class="p-rule is-muted">
+      <div class="row col-medium-2">
+        <div class="col-2 col-small-2">
+          <h3 class="p-text--small-caps u-no-margin--bottom">Forum</h3>
+        </div>
+        <div class="col-4 col-small-2">
+          <p><a href="https://discourse.ubuntu.com/c/lxd/126">Discuss with other users&nbsp;&rsaquo;</a></p>
+        </div>
+      </div>
+      <hr class="p-rule is-muted">
+      <div class="row col-medium-2">
+        <div class="col-2 col-small-2">
+          <h3 class="p-text--small-caps u-no-margin--bottom">Tutorials</h3>
+        </div>
+        <div class="col-4 col-small-2">
+          <p><a href="https://www.youtube.com/c/LXDvideos/videos">Watch tutorials on YouTube&nbsp;&rsaquo;</a></p>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="u-fixed-width p-section--shallow">
+		<hr class="p-rule">
+		<h2>Enhance your experience with <br class="u-hide--small" /> best-in-class open-source tools</h2>
+	</div>
+  <div class="row--25-75">
+    <div class="col">
+      <hr class="p-rule">
+      <div class="row">
+        <div class="col-3 col-medium-2">
+          <h3 class="p-heading--5">Deploy workloads across different platforms with Juju</h3>
+          <p>Juju makes it easy to deploy the exact same software on a variety of different platforms, including public and private cloud, bare metal, Kubernetes clusters. It is also a useful tool for local deployments, ideal for development.</p>
+          <hr class="p-rule">
+          <p>Access the tutorial about <a href="https://juju.is/docs/juju/lxd">operating your instances using Juju&nbsp;&rsaquo;</a></p>
```

Comment:
I did this one on purpose so it can create emphasis, otherwise we have entire sentences of links which I don't think it's good practice? But I don't mind changing it.